dialogue-annotations/src/main/java/com/palantir/dialogue/annotations/InputStreamContentBody.java

```diff
@@ -39,7 +38,7 @@ final class InputStreamContentBody implements ContentBody {
 
     @Override
     public void writeTo(OutputStream output) throws IOException {
-        ByteStreams.copy(inputStream, output);
+        inputStream.transferTo(output);
```

Comment:
Should we get this into error-prone if this is now the preferred approach? I know I'm guilty of many uses of this method

Reply:
good call, I'd like to write up an error-prone rule to migrate folks soon.

Reply:
https://github.com/palantir/gradle-baseline/pull/2616